sale_order_digitized_signature/readme/CREDITS.rst

```diff
@@ -0,0 +1,3 @@
+The development of this module has been financially supported by:
```

Comment:
Uhm, this is not true. We are developers, not funders.

Reply:
@pedrobaeza Who paid the developers?

Reply:
The customer... this is intended for direct customers that want to appear in credits. Tecnativa is already in authors.